### DISCUSSION TITLE
2ο θέμα

### DISCUSSION THREAD

**Discussion by ghost** [sdy60/syllabus#85] [Q&A] [answered]
Στο δεύτερο θέμα χρειαζόσαστε μόνο τα σενάρια ή και μια θεωρητική ανάλυση για κάθε μια από τις 3 περιπτώσεις;

**Comment by epidrome** [comment, marked as answer]
μόνο τα σενάρια, στα οποία θα φαίνονται (έμμεσα) οι περσόνες και η διάδραση τους με το σύστημα
